.github/workflows/build.yml

```
@@ -96,6 +102,14 @@ jobs:
           git push --follow-tags
           echo "commit_hash=$(git rev-parse HEAD)" >> $GITHUB_OUTPUT
 
+      # Upload OpenAPI spec as artifact
+      - name: Upload OpenAPI Specification
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)